### PR TITLE
Add info about using Projects and Boards

### PR DESCRIPTION
**Pull request by amarantha-k**
to plan and track progress for content/documentation updates on xrpl.org.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -105,3 +105,28 @@ $ pybabel compile -f -d ./locale/
 If you added a new language for the first time, you need to make sure its target definition (in the `dactyl-config.yml` file) has the MO file in the `locale_file` field.
 
 After that, next time you build the site using Dactyl it should pull the updated translations!
+
+## Issues, Projects, and Project Boards
+
+Use GitHub Issues to report bugs, feature requests, and suggestions.
```
Comment (mDuo13) [inline README.md]: Maybe clarify that the xrpl-dev-portal repo is for bugs, feature requests, and suggestions _for the docs or website specifically_. It's not uncommon for us to get people reporting issues with the rippled server or client libraries here.

File: README.md [hunk 2]
```diff
@@ -105,3 +105,28 @@ $ pybabel compile -f -d ./locale/
...
+
+## Issues, Projects, and Project Boards
+
+Use GitHub Issues to report bugs, feature requests, and suggestions. 
+
+If you are a contributor, use GitHub Projects and Project Boards to plan and track updates to xrpl.org. 
+
+### Project Board `xrpl-docs`
+
+The [`xrpl-docs`](https://github.com/orgs/XRPLF/projects/4) Kanban board is used to plan and track updates to the XRP Ledger Documentation. The Board only reflects the status of the Issues and contributors must update the status of an issue as it progresses through different stages.
```
Comment (mDuo13) [inline README.md]: I don't think we need to treat "Board" and "Issues" as proper nouns here. But also, I'm not sure what "The board only reflects the status of the issues" is supposed to mean—as opposed to what? It feels like we can just cut that section out and not lose much.

```suggestion
The [`xrpl-docs`](https://github.com/orgs/XRPLF/projects/4) Kanban board is used to plan and track updates to the XRP Ledger Documentation. Contributors must update the status of an issue as it progresses through different stages. 
```

File: README.md [hunk 3]
```diff
@@ -105,3 +105,28 @@ $ pybabel compile -f -d ./locale/
...
+
+Use GitHub Issues to report bugs, feature requests, and suggestions. 
+
+If you are a contributor, use GitHub Projects and Project Boards to plan and track updates to xrpl.org. 
+
+### Project Board `xrpl-docs`
+
+The [`xrpl-docs`](https://github.com/orgs/XRPLF/projects/4) Kanban board is used to plan and track updates to the XRP Ledger Documentation. The Board only reflects the status of the Issues and contributors must update the status of an issue as it progresses through different stages. 
+
+The `xrpl-docs` board has six columns based on the status of Issues in this repository:
```
Comment (mDuo13) [inline README.md]:
```suggestion
The `xrpl-docs` board has six columns based on the status of issues in this repository:
```

File: README.md [hunk 4]
```diff
@@ -105,3 +105,28 @@ $ pybabel compile -f -d ./locale/
...
+
+* **No Status**: Lists new or existing issues which require their Status to be updated.
```
Comment (mDuo13) [inline README.md]:
```suggestion
* **No Status**: New or existing issues that no one has triaged yet.
```

File: README.md [hunk 5]
```diff
@@ -105,3 +105,28 @@ $ pybabel compile -f -d ./locale/
...
+
+### Project Board `xrpl-docs`
+
+The [`xrpl-docs`](https://github.com/orgs/XRPLF/projects/4) Kanban board is used to plan and track updates to the XRP Ledger Documentation. The Board only reflects the status of the Issues and contributors must update the status of an issue as it progresses through different stages. 
+
+The `xrpl-docs` board has six columns based on the status of Issues in this repository:
+
+* **No Status**: Lists new or existing issues which require their Status to be updated.  
+
+* **Backlog**: Lists issues that have been triaged by repository maintainers or contributors. Ensure that the GitHub Issue contains actionable and helpful information for a contributor to work on addressing the issue.
```
Comment (mDuo13) [inline README.md]:
```suggestion
* **Backlog**: Issues that represent tasks to be done eventually. They should contain actionable and helpful information for a contributor to work on addressing the issue.
```

File: README.md [hunk 6]
```diff
@@ -105,3 +105,28 @@ $ pybabel compile -f -d ./locale/
...
+
+The [`xrpl-docs`](https://github.com/orgs/XRPLF/projects/4) Kanban board is used to plan and track updates to the XRP Ledger Documentation. The Board only reflects the status of the Issues and contributors must update the status of an issue as it progresses through different stages. 
+
+The `xrpl-docs` board has six columns based on the status of Issues in this repository:
+
+* **No Status**: Lists new or existing issues which require their Status to be updated.  
+
+* **Backlog**: Lists issues that have been triaged by repository maintainers or contributors. Ensure that the GitHub Issue contains actionable and helpful information for a contributor to work on addressing the issue. 
+
+* **Planned**: Lists issues that have an assignee and are planned to be addressed in the near future, like 2-4 weeks.
```
Comment (mDuo13) [inline README.md]:
```suggestion
* **Planned**: Issues with assignees who plan to address them in the near future, like 2-4 weeks.
```

File: README.md [hunk 7]
```diff
@@ -105,3 +105,28 @@ $ pybabel compile -f -d ./locale/
...
+
+The `xrpl-docs` board has six columns based on the status of Issues in this repository:
+
+* **No Status**: Lists new or existing issues which require their Status to be updated.  
+
+* **Backlog**: Lists issues that have been triaged by repository maintainers or contributors. Ensure that the GitHub Issue contains actionable and helpful information for a contributor to work on addressing the issue. 
+
+* **Planned**: Lists issues that have an assignee and are planned to be addressed in the near future, like 2-4 weeks. 
+
+* **In Progress**: Lists issues that are in progress.
```
Comment (mDuo13) [inline README.md]:
```suggestion
* **In Progress**: Issues that a contributor is actively working on.
```

File: README.md [hunk 8]
```diff
@@ -105,3 +105,28 @@ $ pybabel compile -f -d ./locale/
...
+
+* **No Status**: Lists new or existing issues which require their Status to be updated.  
+
+* **Backlog**: Lists issues that have been triaged by repository maintainers or contributors. Ensure that the GitHub Issue contains actionable and helpful information for a contributor to work on addressing the issue. 
+
+* **Planned**: Lists issues that have an assignee and are planned to be addressed in the near future, like 2-4 weeks. 
+
+* **In Progress**: Lists issues that are in progress. 
+
+* **In Review**: Lists issues that are being reviewed.
```
Comment (mDuo13) [inline README.md]:
```suggestion
* **In Review**: Issues with a proposed fix that is currently being reviewed. These should be associated with an open pull request.
```

File: README.md [hunk 9]
```diff
@@ -105,3 +105,28 @@ $ pybabel compile -f -d ./locale/
...
+
+* **Backlog**: Lists issues that have been triaged by repository maintainers or contributors. Ensure that the GitHub Issue contains actionable and helpful information for a contributor to work on addressing the issue. 
+
+* **Planned**: Lists issues that have an assignee and are planned to be addressed in the near future, like 2-4 weeks. 
+
+* **In Progress**: Lists issues that are in progress. 
+
+* **In Review**: Lists issues that are being reviewed. 
+
+* **Done**: Lists issues that have been completed and the related content updates have been merged.
```
Comment (mDuo13) [inline README.md]:
```suggestion
* **Done**: Issues that have been completed, whose related content updates have been merged. 
```

**Review by mDuo13 — APPROVED** [review]
Looks good. I suggested a few edits for clarity.

**Review by Silkjaer — APPROVED** [review]
With the changes @mDuo13 suggest, I think it's great!

File: README.md (hunk 1 above)
Comment (intelliot) [reply README.md]: For those issues, we could point people to https://github.com/XRPLF and/or the rippled, xrpl.js, xrpl-py, et al. repos

**Review by intelliot — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (amarantha-k) [reply README.md]: Incorporated feedback

**Comment by amarantha-k** [comment]
Thanks for the review, @mDuo13, and @intelliot. I've incorporated your feedback.

**Pull request merged**